easybuild/easyconfigs/c/clck/clck-2017.1.016.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = "IntelBase"
+name = 'clck'
+version = '2017.1.016'
+
+homepage = 'https://software.intel.com/en-us/intel-cluster-checker'
+description = """Verifies cluster components work together ― for 
+ - better uptime and productivity and lower total
+ - cost of ownership (TCO)
+ """
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_clck_p_%(version)s.tgz']
+
+checksums = ['84e7c00bf1f9b87e248dd62e4df7e3b7']
+
+dontcreateinstalldir = 'True'
+
+license_file = HOME + '/licenses/intel/license.lic'
+modextrapaths = {
+	'CPLUS_INCLUDE_PATH': 'clck/2017.1.016/include',
+	'LIBRARY_PATH': 'clck/2017.1.016/lib/intel64',
+	'PATH': 'clck/2017.1.016/bin/intel64' 
+}
+
+sanity_check_paths = {
+    'files': ['clck/2017.1.016/bin/clckvars.sh' , 'clck/2017.1.016/bin/clckvars.csh' , 'clck/2017.1.016/bin/intel64/clck-analyze','clck/2017.1.016/bin/intel64/clck-collect', 'clck/2017.1.016/bin/intel64/clckdb'],
```

Comment:
same here, please use `%(version)s` rather than hardcoding the version

also, please wrap this line, lines should not be longer than 120 characters